lite2/client.go

```diff
@@ -705,11 +705,11 @@ func (c *Client) sequence(
 	now time.Time) error {
 	// 1) Verify any intermediate headers.
 	var (
-		interimHeader   *types.SignedHeader
-		interimNextVals *types.ValidatorSet
-		err             error
+		interimHeader *types.SignedHeader
+		err           error
 	)
-	for height := trustedHeader.Height + 1; height < newHeader.Height; height++ {
+	for trustedHeader.Height < newHeader.Height {
```

Comment:
how;s this better than before?

Reply:
it's not - I tried to make it cleaner but there's not really a difference

Reply:
2 lines instead of one

Reply:
Yeah I think reverting it is fine, in my defense, my line of thought was that we might not need to declare height in the for loop but we use it so often so I sort of declared it afterwards 😅